go-samson-secret-puller/secrets.go

```diff
@@ -0,0 +1,36 @@
+package samsonsecretpuller
```

Comment:
Underscores would be un-idiomatic and dashes don't work. I would argue for simply `samson`. Client-code will need to type this prefix all the time (unless aliased), so it's nice to be brief. `samson.Secrets()` reads nicely to me.